README.md

```diff
@@ -144,11 +144,40 @@ To automatically register your metadata into the UnitOfWork, this bundle provide
 At this point your metadata are declared into the UnitOfWork, thanks to a CompilerPass.
 You are ready to upload data.
 
-### Upload data to google bigquery
-To upload data, you need to use the service `CCMBenchmark\BigQueryBundle\BigQuery\UnitOfWork`.
-This service offers a simple API. Call `addData` to store a new Entity to upload.
+### Working with UnitOfWorl service
+You need to use the service `CCMBenchmark\BigQueryBundle\BigQuery\UnitOfWork`.
+This service offers a simple API.
+
+#### Upload data to google bigquery
+Call `addData` to store a new Entity to upload.
 When all you're entities are in the UnitOfWork, call `flush` to upload it.
 
+#### Request data from google bigquery
+Call `requestData` to make a request to the specified `projectId` it will return a `\Google\Service\Bigquery\GetQueryResultsResponse`
+```php
+<?php
+class myDataSource implements DataSourceInterface
```

Comment:
`DataSourceInterface` does not belongs to this project, it shoud not be referenced in this example